andela/fengshui-rc
# #150749929 Real time search

#### What does this PR do?
This pull request allows users to be able to search and see the results in real time 

#### Description of Task to be completed?
Similar to a google search, display relevant results on the page as a user is typing rather than wait until they submit the search.

#### How should this be manually tested?
Create an .env  file with the following variables 
- REACTION_EMAIL
- REACTION_AUTH
- REACTION_USER

Ensure that selenium server and chromedriver have been installed then make sure you set your admin details in tests/acceptance-tests/config/user-data.yml file

#### Any background context you want to provide?
Please make sure you follow all the instructions stated above also 
**note** :  `REACTION_EMAIL`, `REACTION_PASSWORD` should be emails and password your generated by `reaction`

#### What are the relevant pivotal tracker stories?
Display real-time results as shopper is typing

#### Screenshots (if appropriate)
<img width="1033" alt="screen shot 2017-09-06 at 7 10 46 pm" src="https://user-images.githubusercontent.com/23409769/30127472-537277c8-9337-11e7-8ac9-9bdba324b1be.png">

#### Questions:
nil
